### DISCUSSION TITLE
Nested Dropdown Menu and overflow = scroll, Please help ;(

### DISCUSSION THREAD

**Discussion by harshRishi** [rsuite/rsuite#3219] [Q&A]
Can anyone explain to me, how we can add scroll to all the nested Dropdown Menu,
Like the way it is shown in this example, [here](https://stackoverflow.com/questions/44396856/css-for-long-scrolling-dropdown-menu)

**Comment by SevenOutman** [comment]
Hi @harshRishi, there's no way to make submenus scroll for now as far as I'm aware.
